src/PrestaShopBundle/Twig/AdminExtension.php

```diff
@@ -23,41 +24,53 @@
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-namespace PrestaShopBundle\Service\Listeners;
 
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+namespace PrestaShopBundle\Twig;
+
 use Symfony\Component\Yaml\Parser;
-use Symfony\Component\Routing\Router;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\ParameterBag;
 
-class AdminControllersListener
+/**
+ * Twig extension for the Symfony Asset component.
```

Comment:
not realy

We should read instead ``Twig extension to share globaly the admin header menu``.